sample_trial/services/application-dev.properties

```diff
@@ -0,0 +1,11 @@
+spring.liquibase.change-log=classpath:/db/changelog/changelog-master.xml
+spring.liquibase.drop-first=true
+spring.liquibase.enabled=true
+spring.jpa.hibernate.ddl-auto=none
```

Comment:
are those really need to be here? 
this is a global setting that will go to all services...

Reply:
@darrengsmithson

Reply:
If the properties are at a 'global' level, they can be put in application.properties (or application-dev, application-int etc) to be picked up by all services, service specific properties are placed in [service]-properties.